include/ui/dockwidget.h

```diff
@@ -90,6 +90,7 @@ class DockWidget : public QDockWidget {
   void splitVertically();
 
  protected:
+  void focusInEvent(QFocusEvent * event) Q_DECL_OVERRIDE;

```

Comment:
it is unifed with other methods in this file so I prefer to not change that, and some day change everything at once